gem/chhaajja/videos/models.py

```diff
@@ -0,0 +1,46 @@
+from django.db import models
+
+from wagtail.core.models import Page
+from wagtail.core.fields import RichTextField
+from wagtail.admin.edit_handlers import FieldPanel
+from wagtail.search import index
+
+class VideosIndexPage(Page):
+    intro = RichTextField(blank=True)
+
+    content_panels = Page.content_panels + [
+        FieldPanel('intro', classname="full")
+    ]
+
+    def get_context(self, request):
+        context = super().get_context(request)
+
+        # Add extra variables and return the updated context
+        context['videos_entries'] = VideosPage.objects.live()
+        context['latest_video'] = VideosPage.objects.latest('id')
+        return context
+
+
+
+
+# Keep the definition of BlogIndexPage, and add:
+
+
+class VideosPage(Page):
+	date = models.DateField("Post date")
```

Comment:
what is the purpose of this date field? Wagtail page models already come with date fields for first published, last published and scheduled publish date.